package.json

```diff
@@ -24,13 +24,21 @@
   ],
   "scripts": {
     "install": "node-gyp-build",
-    "prebuild": "prebuildify --napi --strip --target=10.4.0",
+    "prebuild-single": "prebuildify --napi --strip --target=10.4.0",
```

Comment:
Which builds are we producing here? Are both 32-bit and 64-bit produced on x86 and Arm? Are they all needed?

Reply:
`prebuild-single` builds a single (normal) build
`prebuild-arm64` cross compiles (used for ubuntu)
`prebuild-darwin-x64+arm64` builds a universal binary for mac that includes both x86_64 and arm64